packages/lit-dev-content/site/tutorials/content/svg-templates/01.md

```diff
@@ -0,0 +1,80 @@
+### Overview
+
+This is chapter of the demo covers the basics of using SVG with Lit
```

Comment:
I don't think we should call these `chapters` as it has a really heavy feel.

Reply:
Does 'part' or 'section' have less gravity?

Reply:
i removed chapters from the series